vm/actor_interface/Cargo.toml

```
@@ -20,13 +20,13 @@ anyhow.workspace                 = true
 cid                              = { workspace = true, default-features = false, features = ["std"] }
 forest_encoding.workspace        = true
 forest_fil_types.workspace       = true
-forest_hash_utils.workspace      = true
 forest_ipld_blockstore.workspace = true
 forest_json.workspace            = true
-forest_json_utils.workspace      = true
+forest_utils.workspace           = true
 fvm.workspace                    = true
 fvm_ipld_bitfield.workspace      = true
 fvm_ipld_encoding.workspace      = true
+fvm_ipld_hamt                    = "0.5"
```

`fvm_ipld_hamt` is part of the workspace.

```suggestion
fvm_ipld_hamt.workspace = true
```